addons/resource.language.en_gb/resources/strings.po

```diff
@@ -876,7 +876,9 @@ msgctxt "#203"
 msgid "Plot outline"
 msgstr ""
 
-#empty string with id 204
+msgctxt "#204"
```

Comment:
mind adding a contextual note and also the place where it's used? Something like:
```
#. Name of an add-on type/category
#:  xbmc/addons/Addon.cpp
```
Also, we prefer higher string IDs these days (5 digits), because lower numbers will give way to much wrong hits when searching the code for certain IDs. Thanks.